setup.py

```diff
@@ -31,6 +31,7 @@
     "html5lib>=0.999,<1",
     "Unidecode>=0.04.14",
     "Willow>=0.4,<0.5",
+    "requests>=2.11.1",
```

Comment:
It would probably be better to have `"requests>=2.11.1,<3.0"` here, to guard against the possibility that an incompatible 3.0 version of `requests` is released in future.

(Yes, a few of the other dependencies above also have this problem. We should really fix those too :-) )